src/scripts/OSFramework/OSUI/Pattern/Dropdown/ServerSide/scss/_dropdown-serverside.scss

```diff
@@ -227,7 +228,8 @@ $balloonMobileTopMargin: 5vh;
 	// When it's not valid
 	&--not-valid {
 		.osui-dropdown-serverside__selected-values-wrapper {
-			border-color: var(--color-error);
+			/* !important should be here in order to grant more specific selectors be overriding border color at this context. */
+			border-color: var(--color-error) !important;
```

Comment:
Personally, I would avoid it and just increase specificity if needed, but will approve the PR

Reply:
I understand your point @BenOsodrac, however this is an issue when **has-accessible-features selector** is enabled since the new border color for the A11Y will be more specific which ends on not having the expected border color (not-valid). 
On that case we need to choose between redefine this color when A11Y is enabled, or this one...